spharpy/samplings/coordinates.py

```diff
@@ -310,3 +310,19 @@ def n_max(self, value):
             self._n_max = None
         else:
             self._n_max = int(value)
+
+    def _set_points(self, x, y, z):
+        # set points first to make sure data is valid
+        super()._set_points(x, y, z)
+
+        # check if all points have the same radius
+        radius = self.radius
+        radius_delta = np.max(np.abs(np.mean(radius) - radius))
+        # add a tolerance of 1 mm
+        radius_tolerance = .001
```

Comment:
Using a fixed absolute tolerance is probably not the best idea here.
I would suggest making this relative, or even better make this tolerance a property of the class.

Reply:
we have absolute tolerances in others places in pyfar, so I would prefer to leave it like this. Since its intended to account for numerical noise, I guess it is fine as well.